### PR TITLE
Hack new keywords

### PR DESCRIPTION
**Pull request by ismailsunni**
Try to apply new keywords system to InaSAFE.

Don't merge it yet.

### CODE REVIEW

File: safe/utilities/keyword_io.py [hunk 1]
```diff
@@ -670,3 +678,83 @@ def get_statistics(self, layer):
             statistics_classes = {}
 
         return statistics_type, statistics_classes
+
+
+def write_to_new_keyword(source, keywords):
```
Comment (timlinux) [inline safe/utilities/keyword_io.py]: Can you change these method names to:

write_iso_keywords
read_iso_keywords ?

Comment (ismailsunni) [inline safe/utilities/keyword_io.py]: Yes, I will. Thanks. I just named it like this to experiment.
There are so many to do, including updating the old data tests.

**Comment by timlinux** [comment]
Just an update that we chatted to Charlotte about this and we are going to wait for 3.3 to add this work...

**Comment by timlinux** [comment]
Closing this as stale

**Pull request closed without merging**